src/templates/antora/util.ts

```diff
@@ -0,0 +1,252 @@
+import * as fs from 'fs';
+import * as Handlebars from 'handlebars';
+import * as YAML from 'js-yaml';
+import * as path from 'path';
+
+import { ExportStructure } from '../../lib/exporter';
+import { Type } from '../../lib/type';
+
+interface NavFile {
+  readonly title: string;
+  readonly path: string;
+  readonly pages?: NavFile[];
+}
+
+export interface PageOutput {
+  readonly type: 'adoc' | 'json' | 'yml';
```

Comment:
Is this correct? It should only be able to output `json` and `adoc` to my knowledge.

Reply:
Yep, reverting the change.